chart/values.yaml

```diff
@@ -27,6 +27,7 @@ remoteControlPlane:
   topicSubscription: "sh.keptn.event.test.triggered,sh.keptn.event.deployment.triggered,sh.keptn.event.evaluation.triggered,sh.keptn.event.release.triggered,sh.keptn.event.approval.triggered" # Specify list of CloudEvents to subscribe to in the case of remote control plane
   api:
     protocol: "http"                         # Used Protocol (http, https)
+    # port: 80                               # (Optional) If the API is reachable over a non-standard port
     hostname: "api-gateway-nginx.keptn"      # Hostname of the control plane cluster (and Port)
```

Comment:
In the case of adding port as a value, the documentation of `hostname` does not work (says "and Port")

Reply:
I reverted that commit, I think it's better to stay with the old solution by just adding the Port to the `hostname`